draft/Contest_API.md

```diff
@@ -1084,44 +1084,96 @@ Returned data:
 | :---------- | :------------- | :-------- | :-------- | :----------
-| id          | ID             | yes       | no        | Identifier of the team-member.
+| id          | ID             | yes       | no        | Identifier of the person.
 | icpc\_id    | string         | no        | yes       | External identifier from ICPC CMS.
-| team\_id    | ID             | yes       | no        | [Team](#teams) of this team member.
-| name        | string         | yes       | no        | Name of team member.
-| email       | string         | no        | yes       | Email of team member.
+| team\_id    | ID             | no        | yes       | [Team](#teams) of this person. Required iff role is `team`.
+| name        | string         | yes       | no        | Name of the person.
+| title       | string         | no        | yes       | Title of the person, e.g. "Technical director".
+| email       | string         | no        | yes       | Email of the person.
 | sex         | string         | no        | yes       | Either `male` or `female`, or possibly `null`.
-| role        | string         | yes       | no        | One of `contestant` or `coach`.
-| photo       | array of IMAGE | no        | yes       | Registration photo of the team member.
+| role        | string         | yes       | no        | One of `contestant`, `coach`, or `staff`.
+| photo       | array of IMAGE | no        | yes       | Registration photo of the person.
 
 #### Examples
 
 Request:
 
-` GET https://example.com/api/contests/wf14/team-members`
+` GET https://example.com/api/contests/wf14/people`
 
 Returned data:
 
 ```json
 [{"id":"john-smith","team_id":"43","icpc_id":"32442","name":"John Smith","email":"john.smith@kmail.com","sex":"male","role":"contestant"},
- {"id":"osten-umlautsen","team_id":"43","icpc_id":null,"name":"Östen Ümlautsen","sex":null,"role":"coach"}
+ {"id":"osten-umlautsen","team_id":"43","icpc_id":null,"name":"Östen Ümlautsen","sex":null,"role":"coach"},
+ {"id":"bill","name":"Bill Farrell","sex":"male","title":"Executive Director","role":"staff"}
+]
+```
+
+### Accounts
+
+The accounts used for access to the contest.
+
+The following endpoints are associated with accounts:
+
+| Endpoint                       | Mime-type        | Required? | Description
+| :----------------------------- | :--------------- | :-------- | :----------
+| `/contests/<id>/accounts`      | application/json | yes       | JSON array of all accounts with elements as defined in the table below.
+| `/contests/<id>/accounts/<id>` | application/json | yes       | JSON object of a single account with elements as defined in the table below.
+
+JSON elements of problem objects:
+
+| Name              | Type    | Required? | Nullable? | Description
+| :---------------- | :------ | :-------- | :-------- | :----------
+| id                | ID      | yes       | no        | Identifier of the account.
+| user              | string  | yes       | no        | The account username.
```

Comment:
I really don't like this. If it exists it should be `username` and not `user`, but I would rather have `id` of type `string` instead.

Reply:
But then if we have an archive in CAF format and we later add `profile_picture`, where do we store it? Clearly we can't do `<contestid>/accounts/<id>/profile-picture.png` since the ID can be `/../../../../usr/bin/sudo`